### ISSUE TITLE
self-supervised learning to train a teacher model and to generate pseudo-labels

### ISSUE THREAD

**Issue by Xglbrilliant** [szq0214/FKD#6]
Hi author, thank you for your great work! I am intrigued by your work and very much looking forward to your method for self-supervised learning to train a teacher model and to generate pseudo-labels, thanks a lot!

**Comment by szq0214** [comment]
Hi @Xglbrilliant, thanks for your interest in our work! Our FKD-SSL is based on our previous work: [S2-BNN: Bridging the Gap Between Self-Supervised Real and 1-bit Neural Networks via Guided Distribution Calibration (CVPR 2021)](https://openaccess.thecvf.com/content/CVPR2021/papers/Shen_S2-BNN_Bridging_the_Gap_Between_Self-Supervised_Real_and_1-Bit_Neural_CVPR_2021_paper.pdf) ([Code](https://github.com/szq0214/S2-BNN)), which is a pure distillation-based self-supervised learning framework. The self-supervised teachers can be pre-trained via any strong SSL methods, such as MoCo, SwAV, DINO, etc., or use their off-the-shelf pre-trained models directly.
